src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/SyntaxTriviaListExtensions.vb

```diff
@@ -8,6 +8,15 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
     Friend Module SyntaxTriviaListExtensions
 
+        <Extension()>
+        Public Function ContainsCommentOrLineContinue(list As IEnumerable(Of SyntaxTrivia)) As Boolean
+            If list Is Nothing Then
+                Return False
+            End If
+
+            Return list.Any(Function(t) t.IsKind(SyntaxKind.LineContinuationTrivia) OrElse t.IsRegularOrDocComment)
```

Comment:
Impl doesn't match name. It also considers doc comments. Update make accordingly, or fix code to match.

Reply:
I this case I don't need to know about Doc Comments because they can't get her but as a generic function used in other places to remove "Private" for example you need to consider DocComments. For this function any comment returns true.

Reply:
Update name accordingly, or fix code to match.

Reply:
update name accordingly please.